src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/ShiftService.kt

```diff
@@ -26,63 +24,85 @@ class ShiftService(private val prisonService: PrisonService,
         val start = fromParam.orElse(LocalDate.now(clock))
         val end = toParam.orElse(LocalDate.now(clock))
         val region = prisonService.getPrisonForUser()?.region
-        val details = csrClient.getDetailsForUser(start, end, region, authenticationFacade.currentUsername)
-        val detailsByDate = groupDetailsByDate(details)
+        val detailsByDate = groupDetailsByDate(start, end, region)
 
         return start.datesUntil(end.plusDays(1)).map { date ->
```

Comment:
Why are we doing `end.plusDays(1)` here?

Reply:
I think it needs a comment, its because datesUntil is inclusive of the first date but exclusive of the last date, so we have to +1 to process our entire range.